dependency-check-suppressions.xml

```diff
@@ -24,5 +24,6 @@
         <cve>CVE-2020-8022</cve>
         <cve>CVE-2020-24616</cve>
         <cve>CVE-2017-18640</cve>
+        <cve>CVE-2020-5421</cve>
```

Comment:
Please use <note> to point out Libraries transitively dependent on Current SpringBoot Version